forum/notifications/models.py

```diff
@@ -1,3 +1,18 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class NotificationType(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    description = models.TextField()
+
```

Comment:
The description field is optional. If so, add the parameter blank=True

description = models.TextField(blank=True)